skimage/measure/_regionprops.py

```diff
@@ -166,6 +167,23 @@ def extent(self):
         return self.moments[0, 0] / (rows * cols)
 
     @property
+    def feret_diameter(self):
+        from ..morphology.convex_hull import convex_hull_image
+        from ._find_contours import find_contours
+        from scipy.spatial.distance import pdist
+        
+        label_image = self._label_image
+        label = self.label
+        
+        max_feret = 0.0
+        identity_convex_hull = convex_hull_image(label_image == label)
+        coordinates = np.vstack(find_contours(identity_convex_hull, 0.5, 
+                                              fully_connected = 'high'))
+        distances = pdist(coordinates, 'euclidean')
```

Comment:
Do you mean faster in the sense that only one square root will need to be calculated rather than the square roots of all computed distances as happens with `pdist(coordinates, 'euclidean')`?